### PR TITLE
Add release note and update version numbers for v1.5

### PR DESCRIPTION
**Pull request by liuzhe-lz**

### CODE REVIEW

**Comment by liuzhe-lz** [comment]
Some links will become available after merging branch `v1.5` to `master`.

File: docs/en_US/Release.md [hunk 1]
```diff
@@ -1,5 +1,46 @@
 # ChangeLog
 
+## Release 1.5 - 4/13/2020
+
+### New Features and Documentation
+
+#### Hyper-Parameter Optimizing
+
+* New tuner: [Population Based Training (PBT)](https://github.com/microsoft/nni/blob/master/docs/en_US/Tuner/PBTTuner.md)
+* Trials can now report infinity, NaN, and string metrics
```
Comment (QuanluZhang) [inline docs/en_US/Release.md:10]: what does `string metrics` mean?

File: docs/en_US/Release.md [hunk 2]
```diff
@@ -1,5 +1,46 @@
...
+
+* New Pruner: [GradientRankFilterPruner](https://github.com/microsoft/nni/blob/master/docs/en_US/Compressor/Pruner.md#gradientrankfilterpruner)
+* Compressors will validate configuration by default
+* Model compression examples are refactored and improved
+* Added documentation for [implementing compressing algorithm](https://github.com/microsoft/nni/blob/master/docs/en_US/Compressor/Framework.md)
+
+#### Training Service
+
+* Kubeflow now supports pytorchjob crd v1 (thanks @jiapinai)
+* Experimental DLTS support
```
Comment (QuanluZhang) [inline docs/en_US/Release.md:27]: link the doc file

File: docs/en_US/Release.md [hunk 3]
```diff
@@ -1,5 +1,46 @@
...
+### Fixed Bugs
+
+* ENAS cannot have more than one LSTM layers (thanks @marsggbo)
+* NNI manager's timers will never unsubscribe (thanks @guilhermehn)
+* NNI manager may exhaust head memory (thanks @Sundrops)
+* Batch tuner does not support customized trials (#2075)
+* Experiment cannot be killed if it failed on start (#2080)
+* A bug in lottery ticket pruner
+* Some glitches in web UI
+* And more
```
Comment (QuanluZhang) [inline docs/en_US/Release.md:42]: and more?

File: docs/en_US/Release.md (hunk 1 above)
Comment (liuzhe-lz) [reply docs/en_US/Release.md:10]: Metrics of string type. Maybe "trials can now report infinity and NaN as result, as well as metrics in string format"?

File: docs/en_US/Release.md (hunk 3 above)
Comment (liuzhe-lz) [reply docs/en_US/Release.md:42]: "And more!"

File: docs/en_US/Release.md [hunk 4]
```diff
@@ -1,5 +1,46 @@
...
+#### Model Compression
+
+* New Pruner: [GradientRankFilterPruner](https://github.com/microsoft/nni/blob/master/docs/en_US/Compressor/Pruner.md#gradientrankfilterpruner)
+* Compressors will validate configuration by default
+* Model compression examples are refactored and improved
+* Added documentation for [implementing compressing algorithm](https://github.com/microsoft/nni/blob/master/docs/en_US/Compressor/Framework.md)
+
+#### Training Service
+
+* Kubeflow now supports pytorchjob crd v1 (thanks @jiapinai)
```
Comment (scarlett2018) [inline docs/en_US/Release.md:26]: * Kubeflow now supports pytorchjob crd v1 (thanks external contributor @jiapinai )

File: docs/en_US/Release.md [hunk 5]
```diff
@@ -1,5 +1,46 @@
...
+* Experimental DLTS support
+
+#### Overall Documentation Improvement
+
+* Documentation is significantly improved on grammar, spelling, and wording (thanks @AHartNtkn)
```
Comment (scarlett2018) [inline docs/en_US/Release.md:31]: * Numerous document improvement on grammar, spelling, and wording (thanks external contributor @AHartNtkn)

File: docs/en_US/Release.md (hunk 3 above)
Comment (scarlett2018) [reply docs/en_US/Release.md]: let's all use (thanks external contributor **) format for the notable contributions. For minor fixes, we can use a group thanks like.

> There are many other useful fixes that has been delivered with the release, thanks a lot to all the generous contributors (in alphabetical order): @a , @b, ...

File: docs/en_US/Release.md [hunk 6]
```diff
@@ -1,5 +1,47 @@
 # ChangeLog
 
+## Release 1.5 - 4/13/2020
+
+### New Features and Documentation
+
+#### Hyper-Parameter Optimizing
+
+* New tuner: [Population Based Training (PBT)](https://github.com/microsoft/nni/blob/master/docs/en_US/Tuner/PBTTuner.md)
```
Comment (QuanluZhang) [inline docs/en_US/Release.md:9]: better to use the link like this: https://github.com/microsoft/nni/blob/v1.5/docs/en_US/Tuner/PBTTuner.md

**Pull request merged**